stash.py

```diff
@@ -26,76 +26,82 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
         return document_reference
 
 class HttpStatusCode(Enum):
+    def status_code(self) -> int:
+        return self.value[0]
+
+    def status_message(self) -> str:
+        return self.value[1]
 
     # INFORMATIONAL RESPONSES (100–199)
-    CONTINUE = 100
-    SWITCHING_PROTOCOL = 101
-    PROCESSING = 102
-    EARLY_HINTS = 103
+    CONTINUE =(100, 'Information Response - Continue')
+    SWITCHING_PROTOCOL =(101, 'Information Response -Switching protocol')
+    PROCESSING = (102, 'Information Response - Processing')
+    EARLY_HINTS = (103, 'Information Response - Early hints')
 
     # SUCCESSFUL RESPONSES (200–299)
-    OK = 200
-    CREATED = 201
-    ACCEPTED = 202
-    NON_AUTHORITATIVE_INFORMATION = 203
-    NO_CONTENT = 204
-    RESET_CONTENT = 205
-    PARTIAL_CONTENT = 206
-    MULTI_STATUS = 207
-    ALREADY_REPORTED = 208
-    IM_USED = 226
+    OK = (200,'Successful Responce - Ok')
+    CREATED = (201,'Successful Responce - Created')
+    ACCEPTED = (202,'Successful Responce - Accepted')
+    NON_AUTHORITATIVE_INFORMATION = (203,'Successful Responce - Non authoritative information')
+    NO_CONTENT = (204,'Successful Responce - No content')
+    RESET_CONTENT = (205,'Successful Responce - Reset content')
+    PARTIAL_CONTENT = (206,'Successful Responce - Partial content')
+    MULTI_STATUS = (207,'Successful Responce - Multi status')
+    ALREADY_REPORTED = (208,'Successful Responce - Already reported')
+    IM_USED = (226,'Successful Responce - Im used')
 
     # REDIRECTS (300–399)
-    MULTIPLE_CHOICE = 300
-    MOVED_PERMANENTLY = 301
-    FOUND = 302
-    SEE_OTHER = 303
-    NOT_MODIFIED = 304
-    USE_PROXY = 305
-    UNUSED = 306
-    TEMPORARY_REDIRECT = 307
-    PERMANENT_REDIRECT = 308
+    MULTIPLE_CHOICE = (300,'Redirects -Multiple choice')
```

Comment:
@Vid-27 - Kindly have spaces around the hyphen. Expecting it to be _'Redirects - Multiple Choice'_